### PR TITLE
Pal 425 integration test audit service marshalling

### PR DESCRIPTION
**Pull request by dev930018**

### CODE REVIEW

**Comment by dev930018** [comment]
This appears to be caused by the bug present in the audit service which is fixed in its PAL-425 pull request. This pull on hold until further notice

File: audit-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/audit/mock/AuditServiceMock.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package uk.gov.gchq.palisade.integrationtests.audit.mock;
```
Comment (dev930018) [inline audit-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/audit/mock/AuditServiceMock.java]: @Leg60

File: audit-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/audit/service/AuditEndToEndTest.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
+package uk.gov.gchq.palisade.integrationtests.audit.service;
```
Comment (dev930018) [inline audit-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/audit/service/AuditEndToEndTest.java]: @Leg60

Comment (dev930018) [reply audit-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/audit/service/AuditEndToEndTest.java]: Please also review new directory structure (*-service-test -> *-service-integration)

File: audit-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/audit/service/AuditEndToEndTest.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNotNull;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = AuditApplication.class, webEnvironment = WebEnvironment.DEFINED_PORT)
+public class AuditEndToEndTest extends AuditTestCommon {
```
Comment (Leg60) [inline audit-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/audit/service/AuditEndToEndTest.java]: As discussed please review test name to one that is unique to the test written.
Thanks.

**Comment by dev930018** [comment]
Includes an update of windows hadoop resources to a newer version

File: audit-service-integration/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# Copyright 2019 Crown Copyright
```
Comment (ac74475) [inline audit-service-integration/Dockerfile]:
```suggestion
# Copyright 2020 Crown Copyright
```

File: audit-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/audit/mock/AuditServiceMock.java (hunk 1 above)
Comment (ac74475) [inline audit-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/audit/mock/AuditServiceMock.java]: licence and copyright?

File: audit-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/audit/service/AuditServiceComponentTest.java [hunk 5]
```diff
@@ -0,0 +1,66 @@
+package uk.gov.gchq.palisade.integrationtests.audit.service;
```
Comment (ac74475) [inline audit-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/audit/service/AuditServiceComponentTest.java]: licence and copyright?

File: audit-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/audit/service/AuditTestCommon.java [hunk 6]
```diff
@@ -0,0 +1,92 @@
+package uk.gov.gchq.palisade.integrationtests.audit.service;
```
Comment (ac74475) [inline audit-service-integration/src/test/java/uk/gov/gchq/palisade/integrationtests/audit/service/AuditTestCommon.java]: licence and copyright?

File: pom.xml [hunk 7]
```diff
@@ -17,6 +17,12 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+    <dependencies>
```
Comment (ac74475) [inline pom.xml]: Can w have thee dependencies after the modules section

**Pull request merged**